i18n/de.yaml

```diff
@@ -1,5 +1,5 @@
 paige_breadcrumbs:
-  other: Semmelbrösel
+  other: Breadcrumbs
```

Comment:
Unless Semmelbrösel is wrong, we should keep it. Breadcrumbs isn't German.

Reply:
> Unless Semmelbrösel is wrong, we should keep it. Breadcrumbs isn't German.

`Semmelbrösel` is wrong. Addressed in #70.